test/plugin/base_test.rb

```diff
@@ -612,6 +612,30 @@ def test_structured_payload_json_log_detect_json_container_parsed
       end
   end
 
+  # Verify that when the log has only one effective field (named 'log',
+  # 'message' or 'msg') and the field is in JSON format, the field is parsed as
```

Comment:
Oxford comma, please...

Reply:
I meant the comma before the `or`... The "parsed as JSON and sent as jsonPayload" didn't need a comma.